i18n/es/docusaurus-plugin-content-docs/current/campaign-performance/targeting.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar_position: 1
+---
+
+# Ad targeting
+
+Brave has pioneered a new breed of privacy-respecting targeting that is matched directly on the user’s device (aka “client-side”), without any personal data phoning home to Brave’s servers. To achieve targeting at scale in a privacy-respecting capacity, Brave Ads uses on-device machine learning to anonymously match users to relevant ads. The system is built by leading privacy engineers with a range of cryptographic technologies that ensure that no personally-identifiable data ever leaves the user’s device. Contrary to other major ad tech providers who suck up as much personal data as they can, Brave only ever sees anonymous data.
```

Comment:
I was initially unsure about this text since it's not true that "Brave only ever sees anonymous data", Brave definitely needs to see a user's wallet address for payouts, but the text is under the `Ad targeting` section so I guess it's scoped to that.